hadoop-ozone/interface-storage/src/main/java/org/apache/hadoop/ozone/om/codec/OmKeyInfoCodec.java

```diff
@@ -34,7 +34,7 @@
  * Codec to encode OmKeyInfo as byte array.
  *
  * <p>
- * If the layout version "ozone.om.layout.version" is V1 and
+ * If the layout version "ozone.om.layout.version" is PREFIX and
```

Comment:
Is this `ozone.om.metadata.layout` or `ozone.om.layout.version`

Reply:
@ayushtkn Thanks for pointing out this. Its `ozone.om.metadata.layout` , I will update it.